jetbrains-core/src/software/aws/toolkits/jetbrains/services/clouddebug/actions/PseCliAction.kt

```diff
@@ -188,7 +188,7 @@ abstract class PseCliAction(val project: Project, val actionName: String, privat
                         // If this wasn't run through a node, just redraw the whole tree
                         // Open to suggestions to making this smarter.
                     } else {
-                        AwsResourceCache.getInstance(project).clear()
+                        project.clearResourceCache()
```

Comment:
I don't really understand why we're blowing away the whole cache for this connection.

Reply:
Looking through the code..no clue either. I was just blindly converting and keeping what was already there..

Reply:
Hmmm it seems it was a hack due to resource cache doesnt have the concept of cache "categories" like clear everything ECS related.

Reply:
ye - ok fair enough

another possible future enhancement to the cache but for now the nuclear option seems like the best one

Reply:
It looks like it should be possible to unhack...but needs a large re-write of this class to decouple the logic the explorer tree node..which already has a TODO on it to rewrite it 😞